code/modules/admin/verbs/mapping.dm

```diff
@@ -174,7 +174,7 @@ var/intercom_range_display_status = 0
 		if(isturf(thing))
 			continue
 		count++;
-	usr << "There are [count]/[MAX_FLAG] instances of non-turfs in the world."
+	usr << "There are [count]/65535 instances of non-turfs in the world."
```

Comment:
this is incorrect anyways.

The following are limited to 16,777,216 instances at runtime:

/obj
/mob
/area
/image
Appearances (this is something internal with BYOND)
Unique overlay and underlay lists [This is an internal type called an ID array.]
User defined datums
[Consider turfs unofficially on this list.]